domain/build.gradle

```diff
@@ -12,6 +19,8 @@ android {
     targetSdkVersion 29
     versionCode 1
     versionName "1.0"
+    multiDexEnabled true
+    testInstrumentationRunner "org.oppia.android.testing.OppiaTestRunner"
```

Comment:
We don't have instrumentation tests for domain--why is this needed?

Reply:
It is actually not necessary